CHANGELOG.md

```diff
@@ -2,6 +2,29 @@
 
 All notable changes to this project will be documented in this file.
 
+## [1.0.37] - 2023-08-09
+
+### 🚀 New Features and Enhancements
+
+- Add action menu button to zoomable plots [#4398](https://github.com/iterative/vscode-dvc/pull/4398) by [@julieg18](https://github.com/julieg18)
+
+### 🐛 Bug Fixes
+
+- Fix sorts and filters for dep columns [#4432](https://github.com/iterative/vscode-dvc/pull/4432) by [@mattseddon](https://github.com/mattseddon)
+- Exclude untracked dvc.yaml and dvc.lock files from the DVC SCM tree [#4446](https://github.com/iterative/vscode-dvc/pull/4446) by [@mattseddon](https://github.com/mattseddon)
+- Fix Created column presence in show only changed [#4435](https://github.com/iterative/vscode-dvc/pull/4435) by [@mattseddon](https://github.com/mattseddon)
+- Fix broken table when no branches available (detached HEAD on checkout) [#4426](https://github.com/iterative/vscode-dvc/pull/4426) by [@julieg18](https://github.com/julieg18)
+
+### 🔨 Maintenance
+
+- Update demo project and latest tested CLI version (3.13.2) [#4439](https://github.com/iterative/vscode-dvc/pull/4439) by [@mattseddon](https://github.com/mattseddon)
```

Comment:
Weird, most of these prs were released in `1.0.36`. Going to close this and retry running the action.